### PR TITLE
Conda-forge Activity

### PR DESCRIPTION
**Pull request by scopatz**
This was about as annoying as you'd expect

### CODE REVIEW

**Comment by asmeurer** [comment]
Does this do a PR from a fork? Unless it's changed, that's important because otherwise the package deploys from the branch before the PR is merged.

**Comment by scopatz** [comment]
Oh damn, you are right. Uggh that is so annoying.

File: rever/activities/conda_forge.xsh [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+        # lastly make a PR for the feedstock
+        if not pull_request:
+            return
+        gh = github.login()
+        feedstock_reponame = feedstock_repo(feedstock)
+        repo = gh.repository('conda-forge', feedstock_reponame)
+        print('Creating conda-forge feedstock pull request...', file=sys.stderr)
+        title = $PROJECT + ' v' + ver
+        head = 'conda-forge:' + $VERSION
+        body = 'Merge only after success.'
```
Comment (CJ-Wright) [inline rever/activities/conda_forge.xsh]: You might put in something about this being auto-generated by `rever`, could be good PR.

**Comment by asmeurer** [comment]
Relevant upstream issue: https://github.com/conda-forge/conda-forge.github.io/issues/87

**Comment by CJ-Wright** [comment]
What is the workflow for changing the build/run/tests requirements?

**Comment by asmeurer** [comment]
I guess you need some way to know the runner's GitHub username (does rever already have a global way of handling that?), but after that, it's just a matter of adding the fork as a remote and pushing to that remote instead of origin. 

I guess you could also have some code to do the actual forking, but that's optional since it's a do-once thing that most releasers probably have already done.

**Comment by asmeurer** [comment]
@CJ-Wright since the PR is just a normal pull request, you can always do some manual intervention to do other changes to the recipe as necessary. The activity could perhaps print some messages to that effect.

**Comment by scopatz** [comment]
> I guess you need some way to know the runner's GitHub username (does rever already have a global way of handling that?)

Not yet, but I'll add it.

**Comment by scopatz** [comment]
OK! I have updated this to issue the PR from the fork.  This is what the original xonsh release code did, and I was silly to try to change it.

**Comment by scopatz** [comment]
Yeah @CJ-Wright, I agree with @asmeurer.  Major changes to the recipe likely require manual intervention. This is for the business-as-usual case.

**Comment by CJ-Wright** [comment]
Ok.

**Comment by scopatz** [comment]
Note that isn't saying we couldn't add something fancy like that in the future, but it isn't a first-cut feature.

**Comment by CJ-Wright** [comment]
That's fair. That may require a bit more management of the requirements and tests inside the activity.

**Comment by scopatz** [comment]
Thanks @asmeurer and @CJ-Wright for the review!

**Pull request merged**